docs/UpdatingMajorVersionAndTFM.md

```diff
@@ -47,7 +47,11 @@ Once dotnet/runtime has updated their TFM, we update ours in the dependency upda
     * Do not merge this until the PR from the previous step is merged.
 * Update template precedence
   1. Create & merge a PR like [this one](https://github.com/dotnet/spa-templates/pull/39) in dotnet/spa-templates updating `precedence` and `identity` elements in all template.json files.
+      * Going forward, update precedence values by increasing them to the next integer value whose first digits correspond with the major version. This makes it easier to verify that precedence values for any given template were updated since the previous release.
```

Comment:
We probably want to move this do a different section of the doc - this is in the "Updating TFM" section, which we haven't done yet, so if we wait until we update the TFM again next year it might be too late again.

Reply:
Won't the TFM update happen soon™️ and not sometime next year? Or is the idea that we instead should include these template updates with the major version update so we're less likely to forget?

Reply:
Oh, I'm misremembering - the problem you fixed was that we didn't update all of the precedences in the 7.0 era. So it should be fine to do it at TFM time since it technically isn't needed until right before RTM